app/controllers/teams_controller.rb

```diff
@@ -87,7 +87,6 @@ def team_params
       :name, :twitter_handle, :github_handle, :description, :post_info, :event_id,
       :checked, :'starts_on(1i)', :'starts_on(2i)', :'starts_on(3i)',
       :'finishes_on(1i)', :'finishes_on(2i)', :'finishes_on(3i)', :invisible,
-      :project_name,
```

Comment:
Exactly ✅ 

I don't know how the `project_id` param (and the field in the form) actually ended up in the `TeamsController`. It would basically allow teams to change the project they're working on once they have been selected 😹

Reply:
LOL Probably I am to blame for that, when I changed `project_name` to `project_id` 🙈